src/index.js

```diff
@@ -4,58 +4,51 @@ import './index.css';
 
 function Square(props) {
 
-  if (props.value === 'X') {
     return (
       <button 
-        className={"square X " + (props.Winning ? "victory-square" : null)} 
+        className={"square " + (props.Winning ? "victory-square " : null) + (props.value === 'X' ? " X" : " O")} 
         onClick={props.onClick}
       >
         {props.value}
       </button>
     );
-  } else {
-    return (
-      <button 
-        className={"square O " + (props.Winning ? "victory-square" : null)} 
-        onClick={props.onClick}
-      >
-        {props.value}
-      </button>
-    );
-  }
 }
 
 //Board
 
 class Board extends React.Component {
```

Comment:
After applying changes from the bellow comments it'll turn out that you don't need class component but rather a function component (which you should prefer).

Class components are useful if there is something that depends on the instance context (this). For every other case prefer function components.

In general, modern web frameworks prefer functional approach rather than OOP. It goes for both backend and frontend. Functions are first class citizens. Functional approach is easier to read because there is no hidden dependency with a context like methods have. Everything function uses comes from the argument and it makes it much more easy to read and maintain.

Reply:
That makes a lot of sense, much appreciated once again.
Made Board a function.